go-migrate/migrate.go

```diff
@@ -29,6 +29,13 @@ type Migration interface {
 	Revert(Options) error
 }
 
+type SupportNoRevert interface {
```

Comment:
@whyrusleeping I only did this becuase the readme says:

> Frozen. After the tool is written, all code must be frozen and vendored.

I took that that old versions of the conversion tool should not change.  I will happy to backfill it if you think it is better.